genesyscloud/outbound_filespecificationtemplate/resource_genesyscloud_outbound_filespecificationtemplate_utils.go

```diff
@@ -0,0 +1,142 @@
+package outbound_filespecificationtemplate
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v121/platformclientv2"
+)
+
+func buildSdkOutboundFileSpecificationTemplate(d *schema.ResourceData) platformclientv2.Filespecificationtemplate {
+	name := d.Get("name").(string)
+	description := d.Get("description").(string)
+	format := d.Get("format").(string)
+	numberOfHeaderLinesSkipped := d.Get("number_of_header_lines_skipped").(int)
+	numberOfTrailerLinesSkipped := d.Get("number_of_trailer_lines_skipped").(int)
+	header := d.Get("header").(bool)
+	delimiter := d.Get("delimiter").(string)
+	delimiterValue := d.Get("delimiter_value").(string)
+
+	sdkFileSpecificationTemplate := platformclientv2.Filespecificationtemplate{
+		NumberOfHeadingLinesSkipped:  &numberOfHeaderLinesSkipped,
+		NumberOfTrailingLinesSkipped: &numberOfTrailerLinesSkipped,
+		Header:                       &header,
+		ColumnInformation:            buildSdkOutboundFileSpecificationTemplateColumnInformationSlice(d.Get("column_information").([]interface{})),
+		PreprocessingRules:           buildSdkOutboundFileSpecificationTemplatePreprocessingRulesSlice(d.Get("preprocessing_rule").([]interface{})),
+	}
+
+	if name != "" {
```

Comment:
Since name is required you can set it directly on the object like this: 
```
Name: platformclientv2.String(d.Get("name").(string))
```
Same for format